rotkehlchen/api/v1/schemas.py

```diff
@@ -1077,16 +1071,14 @@ class BlockchainAccountDataSchema(Schema):
     tags = fields.List(fields.String(), load_default=None)
 
 
-class BaseXpubSchema(Schema):
+class BaseXpubSchema(AsyncQueryArgumentSchema):
```

Comment:
same as above. This is used here: 

https://github.com/rotki/rotki/blob/98f49cd3ed26c641fec03b78eff9fe1872385fbf/rotkehlchen/api/v1/resources.py#L1117

Also wonder why tests did not catch this. Find out why.